pkg/monitoring/observability/rules/alerts/cluster_alerts.go

```diff
@@ -116,5 +116,22 @@ func clusterAlerts() []promv1.Rule {
 				"operator_health_impact": "none",
 			},
 		},
+		{
+			Alert: "DuplicateWaspAgentDSDetected",
+			Expr: intstr.FromString(
+				`count(kube_daemonset_metadata_generation{namespace="wasp",daemonset="wasp-agent"}) > 0
+					and kubevirt_hco_memory_overcommit_percentage > 100
+			`),
+			For: ptr.To[promv1.Duration]("1m"),
+			Annotations: map[string]string{
+				"summary":     "Duplicate wasp-agent deployment detected",
+				"description": "In the current situation two wasp-agent deployments exist in the same time, the deployment in the wasp namespace should be deleted",
+				"runbook_url": "https://raw.githubusercontent.com/openshift/runbooks/refs/heads/master/alerts/openshift-virtualization-operator/DuplicateWaspAgentDSDetected.md",
```

Comment:
This link is incorrect. It should be 
https://github.com/openshift/runbooks/blob/master/alerts/openshift-virtualization-operator/DuplicateWaspAgentDSDetected.md

Reply:
Thanks! Fixed.